### ISSUE TITLE
Add the main title of each page to the HTML title tag

### ISSUE THREAD

**Issue by LaurentGoderre** [wet-boew/web-reporting#23]
E.g. the title for the Add Website page should be: 'Add Website - Web Reporting'
